src/node_api.cc

```diff
@@ -772,6 +772,11 @@ napi_status napi_get_last_error_info(napi_env env,
   return napi_ok;
 }
 
+NAPI_NO_RETURN void napi_fatal_error(const char* location,
+                                     const char* message) {
+  node::FatalError(location, message);
```

Comment:
It does, but since it was already included in the file I assumed it wasn't a big deal.  Since this is the v8-specific implementation of N-API, this can still be considered internal (I think).